oxd-gen-client/src/test/java/io/swagger/client/api/Tester.java

```diff
@@ -1,23 +1,38 @@
 package io.swagger.client.api;
 
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
 import io.swagger.client.ApiClient;
+import io.swagger.client.model.GetClientTokenParams;
+import io.swagger.client.model.GetClientTokenResponseData;
+import io.swagger.client.model.RegisterSiteResponseData;
+import org.testng.AssertJUnit;
 
 import java.util.List;
 
 import static org.testng.Assert.assertTrue;
+import static org.testng.AssertJUnit.assertNotNull;
 
 
 /**
  * @author yuriyz
  */
 public class Tester {
 
+    private static String AUTHORIZATION = "";
+    private static String HOST;
+    private static String OP_HOST;
+    private static RegisterSiteResponseData clientData;
```

Comment:
minor but this is setup client data, can we rename it to `setupData` or `setupClient` ?